src/styles/blocks/header.scss

```diff
@@ -0,0 +1,15 @@
+.logo {
+  display: flex;
+  margin: 10px auto;
+  width: 40px;
+  height: 40px;
+}
```

Comment:
if its a separate block it should have a separate file with corresponding name, or just make it as element